mantidproject/mantid
# Fix interface squishing in Indirect Corrections interface

**Description of work.**

This commit adds a scroll bar to the tabs in the indirect corrections interface to prevent squishing in smaller screens.

**To test:**

1. open Indirect Corrections check that the layout of all parts of each tab are clear.
2. change screen resolution and make sure that the interface can still be used in the resolution without being squished.

Fixes #30221

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
